.circleci/config.yml

```diff
@@ -14,18 +16,23 @@ jobs:
             apk --no-progress update
             apk --no-progress add bash ca-certificates
             nix-channel --update
+      - run:
+          name: Bazel linter
+          command: |
+            nix-shell --pure --run 'bazel run //:buildifier'
       - run:
           name: Build
           command: |
-            nix-shell --pure --run "bazel build --jobs=2 //... @haskell_zlib//... --config=ci"
-            nix-shell --pure --run "bazel build -c dbg --jobs=2 //... --config=ci"
+            nix-shell --pure --run 'bazel build --jobs=2 //... @haskell_zlib//... --config=ci'
```

Comment:
Why the quotes changes?

Reply:
Some used `'` and some `"`, so I unified them.